### PR TITLE
Fix formatting & add endpoints to refereneces

### PR DESCRIPTION
**Pull request by TGlide**

### CODE REVIEW

File: src/lib/utils/specs.ts [hunk 1]
```diff
@@ -2,310 +2,321 @@ import { OpenAPIV3 } from 'openapi-types';
...
+     * Exceptions for Android SDK.
+     */
+    const isAndroidJava =
+        platform === Platform.ClientAndroidJava || platform === Platform.ServerJava;
+    const isAndroidKotlin =
+        platform === Platform.ClientAndroidKotlin || platform === Platform.ServerKotlin;
+    const isAndroid = isAndroidJava || isAndroidKotlin;
+    const isAndroidServer = platform === Platform.ServerJava || platform === Platform.ServerKotlin;
+    const api = await getApi(version, platform);
+    console.log({ api });
```
Comment (TorstenDittmann) [inline src/lib/utils/specs.ts:219]: leftover

File: src/lib/utils/specs.ts [hunk 2]
```diff
@@ -2,310 +2,321 @@ import { OpenAPIV3 } from 'openapi-types';
...
-							const schema = resolveReference(schemas as OpenAPIV3.ReferenceObject, api);
-							models.push({
-								id,
-								name: schema?.description ?? ''
-							});
-						}
-					}
-				}
+    for (const { method, value, url } of iterateAllMethods(api, service)) {
+        // console.log({ method, value });
```
Comment (TorstenDittmann) [inline src/lib/utils/specs.ts:237]: leftover

**Review by TorstenDittmann — APPROVED** [review]
Please make sure it goes through a design review 👍🏻

**Comment by eldadfux** [comment]
Reminder to add some padding between the sticky top border and the `path` code block

**Comment by eldadfux** [comment]
We're also missing the `/v1/` prefix for the paths.

**Review by gewenyu99 — APPROVED** [review]
Nice <3

Maybe address should be on the left tbh, let's talk about this with Chen later.

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
@TGlide Check Asana, there are some changes Chen asked for :)

**Comment by gewenyu99** [comment]
image
One last comment, can we make the copy button here select only the path not the method?

Copy `/getAccount` vs `GET /getAccount` as a figurative example.

**Review by gewenyu99 — APPROVED** [review]
Other than my comment about copy, if that's difficult/not possible right now, we can merge this as is.

**Comment by gewenyu99** [comment]
LGTM, paste is fixed

**Pull request merged**